src/Avalonia.Themes.Default/GroupBox.xaml

```diff
@@ -0,0 +1,60 @@
+﻿<Styles
+    xmlns="https://github.com/avaloniaui"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    >
+
+  <Styles.Resources>
+    <Thickness x:Key="GroupBoxHeaderThemePadding">4</Thickness>
+  </Styles.Resources>
+  
+  <Style Selector="GroupBox">
+    <Setter Property="Background" Value="{DynamicResource ThemeAccentBrush}" />
```

Comment:
Might need to make IsFocusable and IsTapStop false

I beleive WPF version doest this:
https://github.com/dotnet/wpf/blob/master/src/Microsoft.DotNet.Wpf/src/PresentationFramework/System/Windows/Controls/GroupBox.cs#L24

Reply:
Ok. I will add it.

Reply:
Override some metadata:
Focusable: false
TabNavigation: None